.github/workflows/lint.yaml

```diff
@@ -13,12 +13,13 @@ jobs:
     name: Lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Setup node
         uses: actions/setup-node@v3
         with:
           node-version: 16
+          cache: yarn
```

Comment:
I added this, but perhaps you should revisit the manual caching you are doing in some steps @yisibl (it might not be needed) :)